docs/_adapters/app-mesh.md

```diff
@@ -11,6 +11,23 @@ image: /assets/img/service-meshes/app-mesh.svg
 permalink: service-meshes/adapters/app-mesh
 ---
 
+{% assign sorted_tests_group = site.compatibility | group_by: "meshery-component" %}
+{% for group in sorted_tests_group %}
+      {% if group.name == "meshery-app-mesh" %}
+        {% assign items = group.items | sort: "meshery-component-version" | reverse %}
+        {% for item in items %}
+          {% if item.meshery-component-version != "edge" %}
+            {% if item.overall-status == "passing" %}
+              {% assign adapter_version_dynamic = item.meshery-component-version %}
+              {% break %}
+            {% elsif item.overall-status == "failing" %}
+              {% continue %}
+            {% endif %}
+          {% endif %}
+        {% endfor %} 
+      {% endif %}
+{% endfor %}
+
```

Comment:
I'm not able to see the version here...

![image](https://user-images.githubusercontent.com/59508546/208636399-6b8486d0-fa17-4594-9a7c-b8c4a932ab09.png)

Is this intended?

Reply:
> I'm not able to see the version here...
> 
> ![image](https://user-images.githubusercontent.com/59508546/208636399-6b8486d0-fa17-4594-9a7c-b8c4a932ab09.png)
> 
> Is this intended?

The reason it is not visible for App mesh because it doesn't has `adapter version` defined [here](https://github.com/meshery/meshery/blob/master/docs/_adapters/app-mesh.md). What im trying to do is source its adapter version from its folder in [_compatibilty](https://github.com/meshery/meshery/tree/master/docs/_compatibility).
Since there is no corresponding folder **app-mesh** in _compatibilty ,it can't source the `meshery-component-version` which the code is trying to do. The reason i have made the code changes for this adapter so that whenever a folder **app-mesh** is created in [_compatibilty](https://github.com/meshery/meshery/tree/master/docs/_compatibility) which has files in which `meshery-component-version` is defined ,the website will automatically source this dynamic adapter version.

Reply:
I see... thanks for notif...